Tasks/Kuchinskiy/Task7/Task5.py

```diff
@@ -0,0 +1,25 @@
+import pickle
+
+try:
+    with open("dic.pickle", 'rb') as f:
+        dic = pickle.load(f)
+except FileNotFoundError:
+    dic = {}
```

Comment:
it's a good way to store your cache between sessions, but on the other hand it will be reused by each and every decorated function